src/Common/ProgressIndication.h

```diff
@@ -93,7 +90,7 @@ class ProgressIndication
 
     bool write_progress_on_update = false;
 
-    std::unordered_map<String, UInt64> host_active_cores;
+    std::unordered_map<String, double> host_cpu_usage;
```

Comment:
Why double? It seems for me strange to see non-integral number of used CPUs

Reply:
For me it's not strange at all :) If we spend half second for calculations on one CPU core and half second for waiting - it will be 0.5 CPU.

Also it looks better. No "bouncing" effect like 0 1 0 1 0...

See also `top`. It has %CPU column that is actually the relative number of CPU cores. E.g. on my machine it will go up to 3200%.

Reply:
And it better reflects the nature of this measurement. It is a fraction.
If we do rounding - the natural sense is lost.